assets/capability/capabilities/alarm_pm01.json

```
@@ -0,0 +1,121 @@
+{
+  "type": "boolean",
+  "title": {
+    "en": "PM0.1 alarm",
+    "nl": "PM0.1 alarm",
```

To keep it inline with `alarm_pm25`
```suggestion
    "nl": "PM0,1 alarm",
```